### PR TITLE
Touchups for nixos/xfce4-14 for 19.09

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Thought to do this after a bit of user feedback in https://github.com/NixOS/nixpkgs/issues/68977

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: nixos/modules/services/x11/desktop-managers/xfce4-14.nix [hunk 1]
```diff
@@ -69,16 +70,18 @@ in
 
       exo
       garcon
-      gtk-xfce-engine
       libxfce4ui
       xfconf
 
       mousepad
+      parole
+      pavucontrol
```
Comment (ivan) [inline nixos/modules/services/x11/desktop-managers/xfce4-14.nix]: Should this be included only if `config.hardware.pulseaudio.enable`?

Comment (worldofpeace) [reply nixos/modules/services/x11/desktop-managers/xfce4-14.nix]: Yes. I wonder what program non pulseaudio users should use.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -22704,7 +22704,6 @@ in
   solarc-gtk-theme = callPackage ../misc/themes/solarc { };
 
   xfce = xfce4-12;
```
Comment (volth) [inline pkgs/top-level/all-packages.nix]: Should we set `xfce = xfce4-14` ?

Comment (worldofpeace) [reply pkgs/top-level/all-packages.nix]: Yeah, but I plan to do that for 20.03 and I'd like to backport this PR.

Comment (volth) [reply pkgs/top-level/all-packages.nix]: Yes, but "for 20.03" is "in current master" because 19.09 is already forked, right?

**Pull request merged**